core/src/main/java/com/graphhopper/config/CustomAreaFile.java

```diff
@@ -0,0 +1,84 @@
+package com.graphhopper.config;
+
+
+public class CustomAreaFile {
```

Comment:
Isn't this more a CustomAreaConfig and why is this separate from CustomArea?

Reply:
Should i merge them and keep `location` in the CustomArea? This field is only used to point to file and is not needed anymore after the border polygons have been loaded in GraphHopperManaged.

Reply:
Removing CustomAreaFile leads to the Dropwizard issue i mentioned [earlier](https://github.com/graphhopper/graphhopper/pull/2201#issuecomment-738796938), except that i can't ignore the `customAreas` field as a workaround anymore.